### PR TITLE
Add dummy db class; update README

### PR DESCRIPTION
**Pull request by braddf**
# Pull Request

## Description

Add workaround for local development without DB

## How Has This Been Tested?

Locally with/without setting the `DB_URL` env variable.

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/database.py [hunk 1]
```diff
@@ -44,7 +45,64 @@
 )
 from utils import filter_forecast_values, floor_30_minutes_dt, get_start_datetime
 
-db_conn = DatabaseConnection(url=os.getenv("DB_URL", "not_set"))
```
Comment (peterdudfield) [inline src/database.py:47]: we probably cant merge this, as we wont have the real data base connection. Was this just to try something and see if it works?

File: src/database.py [hunk 2]
```diff
@@ -44,7 +45,64 @@
...
+    Methods
+    -------
+    get_client : static method
+        It gets the database client. If a valid Postgresql database URL is set in
+        the "DB_URL" environment variable, get_client returns an instance
+        of DatabaseConnection(). If not, it returns an instance of the DummyDBClient.
+    """
+
+    @staticmethod
+    def get_client():
```
Comment (peterdudfield) [inline src/database.py:61]: could we rename name this to `get_connection`, sqlalmacy, seems to ahev `connections` and `clients` so we should try and stick with those

File: src/database.py [hunk 3]
```diff
@@ -44,7 +45,64 @@
...
+    def __init__(self):
+        """Initializes the DummyDBClient"""
+        pass
+
+    def get_session(self):
+        """Returns None for now, but mock the session if we keep the current implementation."""
+        return None
+
+
+def get_db_client() -> BaseDBClient:
```
Comment (peterdudfield) [inline src/database.py:100]: similar to above change to connection

File: src/database.py (hunk 2 above)
Comment (braddf) [reply src/database.py]: Yep, sorry this is a little confused over what is a client and what is a connection... it's basically both atm 😅 have changed

**Review by peterdudfield — APPROVED** [review]

**Pull request merged**